org.mwc.debrief.help/collate_docbook.xml

```diff
@@ -333,18 +328,19 @@
             <arg value="-failOnHTMLError"/>
             <arg value="false"/>
             <arg value="-subpackages"/>
-            <arg value="."/>
+            <arg value="org.mwc.debrief.scripting.wrappers"/>
+            <arg value="-link"/>
+        	<arg value="https://docs.oracle.com/javase/8/docs/api"/>
```

Comment:
@saulhidalgoaular - we are building on Java11 now, does this need to be updated?

Reply:
Fixed :)

Reply:
I have used the offline option, because I didn't find the `package-list` online.